src/Infrastructure/Event/EventHistory.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Infrastructure\Event;
+
+use App\Domain\Event\EventHistoryInterface;
+use Doctrine\DBAL\Driver\Connection;
+use Ramsey\Uuid\Uuid;
+
+class EventHistory implements EventHistoryInterface
```

Comment:
Thinking about wrap broadway event store and implement this features in there. As this is responsibility of the event store and `EventHistory` doesn't convince me yet